docs/md/sink.md

```diff
@@ -0,0 +1,132 @@
+# Sink
+The workload `Sink` models the terminal stage of a processing pipeline. In a 
+typical workload configuration, a `Sink` can be used to write workflow outputs
+to a desired location in the cloud.
+
+## User Guide
+You can configure the type of `Sink` used in your workload by changing the
+`sink` attribute of your workload request.
+
+### Terra Workspace Sink
+You can write workflow outputs to a Terra Workspace using the `Terra Workspace`
+sink. A typical `Terra Workspace` sink configuration in the workload request
+looks like:
+```json
```

Comment:
As a newer person I'd find it helpful in addition to this generic sink configuration to include a specific example which can then be referenced in the subsections below, maybe with screenshots showing the relation to the Terra UI.

Reply:
Ill do a follow-up PR